scripts/drone/steps/lib.star

```diff
@@ -75,28 +75,49 @@ def identify_runner_step(platform = "linux"):
             ],
         }
 
-def clone_enterprise_step(committish = "${DRONE_COMMIT}"):
+def enterprise_setup_step(source = "${DRONE_SOURCE_BRANCH}", location = "grafana-enterprise", canFail = True):
```

Comment:
It seems like the location is always `../grafana-enterprise` so maybe we could get rid of this argument or set the default as `../grafana-enterprise`, WDYT?

Reply:
I think in other places (like in release pipelines) where the "clone_enterprise_step" is used it clones "grafana-enterprise" in the PWD. Specifically in release pipelines where the "grabpl init-enterprise" command is used, like here: https://github.com/grafana/grafana/blob/main/scripts/drone/events/release.star#L274

To avoid having to change all of those as well I set the default to what was there before, at least.

Reply:
ah okay I see what you're saying now. In the "enterprise_setup_step" it's always "../grafana-enterprise". That makes sense and I think that will simplify the script you called out in the other comment.